tests/core/core.js

```diff
@@ -3,6 +3,11 @@ import SearchConfig from '../../src/core/models/searchconfig';
 import Storage from '../../src/core/storage/storage';
 import StorageKeys from '../../src/core/storage/storagekeys';
 
+jest.mock('../../src/core/utils/uuid');
```

Comment:
In his Spellcheck integration test PR, Connor created a `__mocks__` folder for the SDK. You can read a bit about how that works here: https://jestjs.io/docs/manual-mocks. The tl;dr is that there are multiple ways to manually mock a module. But, we should pick one way and be consistent in our libraries. Since we've already got the `__mocks__` pattern established, let's go with that. I think you'll need to take what you have here and make a `__mocks__/core/utils/uuid` file.